Sources/RoutingKit/TrieRouter.swift

```diff
@@ -208,9 +214,8 @@ extension TrieRouter {
                 desc.append("→ \(name)")
                 desc.append(parameter.description.indented())
             }
-            if let catchall = self.catchall {
+            if let _ = self.catchall {
```

Comment:
This is likely to create a merge conflict with #93

Reply:
I’ll resolve it if it does. This line is unedited in #93 so I assumed there was no conflict.